conversion/conversion.go

```diff
@@ -385,10 +385,10 @@ func dataFromDatabaseForDataflowMigration(targetProfile profiles.TargetProfile,
 	conv.Audit.StreamingStats.ShardToDataflowInfoMap = make(map[string]internal.ShardedDataflowJobResources)
 	conv.Audit.StreamingStats.ShardToGcsResources = make(map[string]internal.GcsResources)
 	conv.Audit.StreamingStats.ShardToMonitoringResourcesMap = make(map[string]internal.MonitoringResources)
-	tableList, err := common.GetIncludedSrcTablesFromConv(conv)
+	tableListWithSchema, err := common.GetIncludedSrcTablesFromConv(conv)
 	if err != nil {
 		fmt.Printf("unable to determine tableList from schema, falling back to full database")
-		tableList = []string{}
+		tableListWithSchema = map[string][]string{}
```

Comment:
I think the following interface will be better - 

```
map[string]SchemaDetails

struct SchemaDetails {
tableDetails TableDetails[]
}

TableDetails struct {
tableName string
}

```

This will help in capturing details at the schema level and future properties which need to be passed at the table level as well.

Note: Please re-use existing structs as appropriate - this is just an indicative guidance.

Reply:
Modified to use structs